projects/igniteui-angular/src/lib/splitter/splitter.component.ts

```diff
@@ -216,8 +216,14 @@ export class IgxSplitterComponent implements AfterContentInit {
     }
 
     public onMoveEnd(delta: number) {
+        const min = parseInt(this.pane.minSize, 10) || 0;
```

Comment:
Should also apply constraints on sibling, otherwise the sibling can be resized beyond its min/max size.

Reply:
@MayaKirova The issue should be resolved now.